src/Controls/tests/TestCases.HostApp/Issues/Issue28986.xaml

```diff
@@ -0,0 +1,120 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Maui.Controls.Sample.Issues.Issue28986"
+             x:Name="TestPage"
+             Title="SafeAreaGuides Test">
+    
+    <Grid x:Name="MainGrid" BackgroundColor="Red">
```

Comment:
If we're going to be modifying the safearea settings on `ContentGrid`

Then we will need to set IgnoreSafeAreas All on this  MainGrid otherwise this Grid will always be insetting the content by the safe area amount